frontend_app_kanban/src/container/Kanban.jsx

```diff
@@ -679,6 +679,7 @@ export class Kanban extends React.Component {
           <KanbanComponent
             config={state.config}
             content={state.content}
+            // End of required props ///////////////////////////////////////////
```

Comment:
What does this comment mean?

Reply:
This line is here to separate required props and optional props to improve code lecture.
I've started writing theses during the big refactoring #6058 which helped me saving time. So I thought it was a good idea to write them when I can to save time for future devs.